spec/rubocop/cop/style/redundant_begin_spec.rb

```diff
@@ -248,6 +248,23 @@ def method
     RUBY
   end
 
+  it 'registers and corrects an offense when using `if` multiple statements in `begin` block' do
```

Comment:
multiple or multi-line? 

Btw, `if` is technically an expression, not a statement (it evaluates to something).

Reply:
Sure! I updated this PR. Thank you!